### PR TITLE
Make preflight error when suite-setup-util fails

### PR DESCRIPTION
**Pull request by NullVoxPopuli**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -34,7 +34,9 @@ jobs:
         run: pnpm lint
       - id: set-matrix
         working-directory: test-packages/support
-        run: echo "matrix=$(node ./suite-setup-util.js --matrix)" >> $GITHUB_OUTPUT
+        run: |
+          matrix_json=$(node ./suite-setup-util --matrix)
```
Comment (NullVoxPopuli) [inline .github/workflows/ci.yml:38]: separating these allows potential errors to halt CI (see first commit's red status)

File: tests/scenarios/engines-test.ts [hunk 2]
```diff
@@ -117,7 +117,7 @@ engineScenarios
   });
 
 engineScenarios
-  .skip('release') // fails due to https://github.com/emberjs/ember.js/pull/20461
+  .skip('release-engines') // fails due to https://github.com/emberjs/ember.js/pull/20461
```
Comment (NullVoxPopuli) [inline tests/scenarios/engines-test.ts:120]: `release` doesn't exist -- but `release-engines` does!

**Comment by NullVoxPopuli** [comment]
All scenarios are green. :tada:

**Comment by ef4** [comment]
Thanks, good fixes.

**Pull request merged**